lib/cfg-grammar.y

```diff
@@ -1104,16 +1104,16 @@ driver_option
     ;
 
 threaded_dest_driver_option
-	: KW_RETRIES '(' LL_NUMBER ')'
+	: KW_RETRIES '(' positive_integer ')'
```

Comment:
This should be `nonnegative_integer` based on this code snippet in logthrdestdrv.c:

```
 if (self->retries.max <= 0)
    {
      msg_warning("Wrong value for retries(), setting to default",
                ...
    }
```

When you try to send the message for the first time, it won't be counted as a "try", so you can specify 0 here.
:-1: